dist/scripts/api-console.js

```diff
@@ -745,7 +745,7 @@
         hidePropertyDetails: '=',
         showExamples: '='
       },
-      controller: function ($scope, $rootScope) {
+      controller: ['$scope', '$rootScope', function ($scope, $rootScope) {
```

Comment:
maintaining this will probably become a nightmare. I suggest you investigate a plugin for the build process called `ng-annotate` that performs this task automatically